### PR TITLE
Add READY_FOR_PICKUP UPR status and misc refactors/bugfixes

### PR DESCRIPTION
**Pull request by ansonjwhe**
- Changed copytext for SF emails
- Replaced placeholder text in SF confirmation modals
- Fixed UPR admin table PO num and Req num states not updating on ticket change
- Fixed claim page crashing (toUpperCase in hooks.js)

### CODE REVIEW

File: backend/service/uwfinancepurchases.service.js [hunk 1]
```diff
@@ -53,6 +54,13 @@ const updateUWFinancePurchase = async (id, body) => {
         ]
         await Promise.all(emails)
     }
+    if (
+        existingPurchaseTicket.status === 'ORDERED' &&
```
Comment (victorzheng02) [inline backend/service/uwfinancepurchases.service.js]: should we do a double check? if so we should probably make it consistent across the other transitions just for cleanliness

File: frontend/src/components/TicketContent/UPRAdminContentTable.js [hunk 2]
```diff
@@ -23,6 +25,23 @@ const UPRAdminContentTable = () => {
         setChanged(true)
     }
 
+    useEffect(() => {
+        if (!currentTicket.requisition_number) {
+            setReqNum('')
+        } else {
+            setReqNum(currentTicket.requisition_number)
```
Comment (victorzheng02) [inline frontend/src/components/TicketContent/UPRAdminContentTable.js]: can't we just use a null coalescer for both of these

**Review by victorzheng02 — APPROVED** [review]
lgtm, two small comments

File: backend/service/uwfinancepurchases.service.js (hunk 1 above)
Comment (ansonjwhe) [reply backend/service/uwfinancepurchases.service.js]: removed for consistency

**Comment by ansonjwhe** [comment]
resolves #179

**Pull request merged**